### PR TITLE
Update Wiki.php

### PR DESCRIPTION
**Pull request by rahafnsr**
upload file with wiki

### CODE REVIEW

File: src/Redmine/Api/Wiki.php [hunk 1]
```diff
@@ -71,6 +71,16 @@ public function create($project, $page, array $params = [])
 
         $xml = new \SimpleXMLElement('<?xml version="1.0"?><wiki_page></wiki_page>');
         foreach ($params as $k => $v) {
+            if ('uploads' === $k && is_array($v)) {
+                 $uploadsItem = $xml->addChild('uploads', '');
+                 $uploadsItem->addAttribute('type', 'array');
+                 foreach ($v as $upload) {
+                     $upload_item = $uploadsItem->addChild('upload', '');
```
Comment (kbsali) [inline src/Redmine/Api/Wiki.php:78]: could you make these snakeCase like the rest of the variables?

File: src/Redmine/Api/Wiki.php [hunk 2]
```diff
@@ -71,6 +71,16 @@ public function create($project, $page, array $params = [])
...
+                     foreach ($upload as $upload_k => $upload_v) {
+                         $upload_item->addChild($upload_k, $upload_v);
+                     }
+                 }
+             } else
             $xml->addChild($k, $v);
```
Comment (kbsali) [inline src/Redmine/Api/Wiki.php:84]: the else case is missing curly brackets `{...}`.
Applying php-cs-fixer should fix the issue

**Comment by rahafnsr** [comment]
Thank you so much!

**Comment by kbsali** [comment]
thanks, although your 2nd commit doesn't fix the code style convention.
I'll merge and fix it after.

**Pull request merged**